portfolio/src/main/webapp/movies.js

```diff
@@ -27,4 +33,34 @@ function showImage(n) {
 
     images[slideIndex - 1].style.display = "block";
     captions[slideIndex - 1].style.display = "block";
+
+    sessionStorage.setItem("currSlide", slideIndex);
+    document.getElementById("curr-movie").value = slideIndex;
+    
+    displayList();
+}
+
+function displayList() {
+    var maxCont = document.getElementById("selectMax");
```

Comment:
Try to use more descriptive variable names. It's not immediately apparent to me what maxCont means.